lib/hammer_cli_katello/lifecycle_environment_name_resolvable.rb

```diff
@@ -1,22 +1,35 @@
 module HammerCLIKatello
   module LifecycleEnvironmentNameResolvable
-    def lifecycle_environment_resolve_options(options)
-      {
-        HammerCLI.option_accessor_name("name") => options['option_environment_name'],
-        HammerCLI.option_accessor_name("id") => options['option_environment_id'],
-        HammerCLI.option_accessor_name("organization_id") => options["option_organization_id"],
-        HammerCLI.option_accessor_name("organization_name") => options["option_organization_name"]
-      }
+    def all_options
+      @all_options ||= super.clone
+
+      @all_options['option_lifecycle_environment_organization_id'] ||= (
+        @all_options['option_organization_id'] ||= resolver.organization_id(
+          resolver.scoped_options('organization', @all_options)
+        )
+      ) if @all_options['option_organization_name']
+
+      @all_options['option_organization_ids'] ||= resolver.organization_ids(
+        resolver.scoped_options('organization', @all_options)
+      ) if @all_options['option_organization_names']
```

Comment:
What about a wrapper If statement instead of a dangling if statement?
 Add a line note